### DISCUSSION TITLE
Is it possible to update AzuraCast retaining the old public page player?

### DISCUSSION THREAD

**Discussion by zcraber** [AzuraCast/AzuraCast#6467] [Q&A] [answered]
![image](https://github.com/AzuraCast/AzuraCast/assets/61133303/e5e02653-d630-4f3c-b26e-718bb9700f8f)

**Comment by BusterNeece** [comment, marked as answer]
Well, no, it's not possible to update AzuraCast while not updating part of AzuraCast.

What's wrong with the new one? You can always use custom CSS to get it looking the way you like...

**Comment by zcraber** [comment]
> What's wrong with the new one?

Nothing wrong, just a matter of taste. Personally, I like the previous player as the new one has a larger progress bar and volume slider.  

![image](https://github.com/AzuraCast/AzuraCast/assets/61133303/fb1b8fa4-7838-47cd-8bd4-741a57d83628)

**Reply by BusterNeece** [reply]
That's fair. I can probably tweak things a little so the public player looks more similar to its predecessor so people aren't so jolted by the change.

**Comment by BusterNeece** [comment]
<img width="571" alt="Screenshot 2023-08-03 110211" src="https://github.com/AzuraCast/AzuraCast/assets/6744885/04a9b8a5-34a2-4e0f-b7d7-077b746fe3ea">

You know, I admit the public player didn't get the same refining pass I applied to a lot of things. I've gone through and fine-tuned it, and I think it looks both more like the original and a lot sharper overall. Let me know what you think.

**Reply by zcraber** [reply]
Looks awesome!
